backend/package.json

```diff
@@ -29,6 +29,7 @@
     "dotenv-safe": "^6.1.0",
     "graphql": "^14.3.0",
     "graphql-yoga": "^1.17.4",
+    "lodash": "^4.17.11",
```

Comment:
We need to update package-lock.json as well since this won't be installed otherwise.